server/models/UserAuthentication.ts

```diff
@@ -50,6 +50,9 @@ class UserAuthentication extends IdModel {
   @Column
   providerId: string;
 
+  @Column
+  oidcTokenId: string;
```

Comment:
I believe this column would need to be encrypted in the database? like refresh and access token